nbconvert/nbconvertapp.py

```diff
@@ -238,21 +238,20 @@ def initialize(self, argv=None):
 
 
     def init_syspath(self):
-        """
-        Add the cwd to the sys.path ($PYTHONPATH)
-        """
+        """Add the cwd to the sys.path ($PYTHONPATH)"""
```

Comment:
In general, I would consider changes like this one purely stylistic cleanup, which we prefer to avoid.